templates/match-detail.html

```diff
@@ -49,4 +49,8 @@ <h2>{{ message }}</h2>
     </div>
 {% endfor %}
 
+<div class="footer">
+    <a class="btn btn-primary btn-lg fixed-bottom" href="javascript:window.history.back();" role="button">Go Back</a>
```

Comment:
On the match detail page, this blocks the last row from displaying

Reply:
Hmmmm. . . guess I should develop on a regular monitor as this is not an issue in 4k.